tests/test_tutorials.py

```diff
@@ -34,13 +30,8 @@ def test_tutorials():
         script_name = notebook_name[:-6] + '.py'
         os.system( 'ipython nbconvert --to=python %s' %notebook_name )
         clean_ipython_features( script_name )
-        try:
-            exec( open(script_name).read() )
-        except:
-            # now we might want to know the script that was executed
-            print( open(script_name).read() )
-            # re-raise same exception to make test fail
-            raise
+        response = os.system('python ' + script_name)
+        assert response==0
```

Comment:
that is great but it might still be helpful to print the actual script on error, too. (because the script is generated on the fly which makes it hard to debug, basically like macro-magic in c code)